### PR TITLE
[SASS-2895] Add employment gateway page

### PR DESCRIPTION
**Pull request by rtna-hmrc**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-2895](https://jira.tools.tax.service.gov.uk/browse/SASS-2895)

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: app/controllers/tailorings/EmploymentGatewayController.scala [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    if(appConfig.tailoringEnabled) {
+      form(request.user.isAgent).bindFromRequest().fold(
+        formWithErrors => Future.successful(BadRequest(view(formWithErrors, taxYear))),
+        yesNo => {
+          if(yesNo) {
+            Future(Redirect(controllers.employment.routes.EmploymentSummaryController.show(taxYear)))
+          }
+          else {
+            Future(Redirect(appConfig.incomeTaxSubmissionOverviewUrl(taxYear)))
+          }
```
Comment (TheDeadlyPianist) [inline app/controllers/tailorings/EmploymentGatewayController.scala:60]: "Yes" should take you to the next part of the journey, "No" should take you to the summary page.

Comment (juamalBlackman) [reply app/controllers/tailorings/EmploymentGatewayController.scala:60]: I thought that employmentSummary was the next part in the journey then from there you are able to select add new employment

**Review by robert-webb-hmrc — CHANGES_REQUESTED** [review]

File: app/views/tailorings/EmploymentGatewayView.scala.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+@errorSummary(form.errors.distinct)
+
+@heading(headingForUse, Some(messages("payeEmployment.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-4")
+
+@formWithCsrf(action = EmploymentGatewayController.submit(taxYear)) {
+@inputRadio(
+form = form,
+legendHeading = Some(Html(headingForUse)),
+legendHidden = true,
+inline = true
```
[the remark this answers is not in the document]
Comment (rtna-hmrc) [reply app/views/tailorings/EmploymentGatewayView.scala.html]: done! Please let me know if this one is fine

File: conf/application.conf [hunk 3]
```diff
@@ -187,6 +187,7 @@ feature-switch {
    nrsEnabled = true
    studentLoans = true
    employmentEOYEnabled = true
+   tailoringEnabled = true
```
[the remark this answers is not in the document]
Comment (rtna-hmrc) [reply conf/application.conf:190]: Yeah it is already merged in the config, thanks!

File: conf/application.conf [hunk 4]
```diff
@@ -177,7 +177,7 @@ accessibility-statement.service-path = "/income-tax-submission"
 
 defaultTaxYear = 2023
 
-timeoutDialogTimeout = 900
+timeoutDialogTimeout = 9000
```
[the remark this answers is not in the document]
Comment (rtna-hmrc) [reply conf/application.conf]: You are right, I missed this one. Just reverted it

**Pull request merged**